src/modules/imageresizer/ui/Models/ResizeOperation.cs

```diff
@@ -105,9 +131,9 @@ public void Execute()
                     encoder.Frames.Add(
                         BitmapFrame.Create(
                             Transform(originalFrame),
-                            thumbnail: null,
+                            originalFrame.Thumbnail,
                             metadata,
-                            colorContexts: null));
+                            originalFrame.ColorContexts));
```

Comment:
Interesting, I took this out a few years ago because it was screwing up the color on some images. See https://github.com/bricelam/ImageResizer/issues/53

Reply:
Ah, looks like you've since taken it out again :-)

Reply:
yeah, stupid me added it because it looked "right". I reverted this change with #14867. Btw: I'm impressed that you remember this issue after all this years